packages/api/src/services/subscriptions.ts

```diff
@@ -199,5 +201,16 @@ export const createSubscription = async (
     unsubscribeMailTo,
     lastFetchedAt: new Date(),
     type: subscriptionType,
+    url,
   })
 }
+
+export const deleteSubscription = async (id: string): Promise<DeleteResult> => {
```

Comment:
Is this new? I thought we were always soft deleting subscriptions now

Reply:
Yeah, this is new and I only use it in the tests for clean-up

Reply:
ohhh haha, great